### ISSUE TITLE
Remove bottom margin  of  the `mj-hero`

### ISSUE THREAD

**Issue by innovaweb-dev** [mjmlio/mjml#1281]
After each `mj-hero`, I have a bottom margin with Miscrosoft Outlook 2016.
And I want to remove this margin.

**Expected behavior:**

Remove bottom margin  of  the `mj-hero`

**Observed behavior:**

There is a bottom margin on the `mj-hero`

**Screenshots and GIFs**

![image](https://user-images.githubusercontent.com/2601347/42714567-f5434df6-86f3-11e8-84cf-9e9c64860fab.png)

**MJML version:**

```html
//Resume
    <mj-body background-color="#e7ecee">
        <mj-hero padding="100px 0px" css-class="margin: 0;" mode="fixed-height" width="600px" height="220px" background-width="600px" background-height="220px" background-url="http://www.domissori.fr/img/mail/email-offer.jpg">
        </mj-hero>
        <mj-section background-color="white" padding="0">

```

**Email clients the bug is seen on:**

Microsoft Outlook 2016

Thank you for your help

**Comment by iRyusa** [comment]
Hi  @rifton007 

Thanks for reporting this 
Tried to fix it this morning but wasn't able to find anything, will check if MJML 3.3.X branch have the same issue

**Comment by rifton007** [comment]
Hi @iRyusa
Had you the same problem ?
May be, have you a temporary solution ?

Thanks,

**Comment by iRyusa** [comment]
Not for now, still trying to reproduce in 3.3.X branch to see if it's a issue introduced in 4

**Comment by kmcb777** [comment]
hi @rifton007 thanks, this will be fixed in v4.2
For now if you can edit the html the fix is to add `top:0;` in the inline-style of the `<v:image` tag
